src/common/Tabs/chooseExperience.tsx

```diff
@@ -207,15 +207,17 @@ const ChooseExperience: React.FC<selectExpModalProps> = ({
                     From landing page <br />
                     search for course,
                   </p>
-                  <p>for example:</p>
-                  <p
-                    style={{
-                      fontWeight: "bolder",
-                      color: "#000",
-                      textTransform: "capitalize",
-                    }}
-                  >
-                    Design{" "}
+                  <p>
+                    e.g. :{""}{" "}
```

Comment:
I think we can remove curly brace?

Reply:
curly braces used for the space between the text bcs